### PR TITLE
Various improvements to `ci-fix.md`

### PR DESCRIPTION
**Pull request by konrad-jamrozik**
This is a PR made by the Azure SDK Engineering System team

### CODE REVIEW

File: documentation/ci-fix.md [hunk 1]
```diff
@@ -1,17 +1,47 @@
...
 This page provides detailed instructions on how to diagnose, reproduce, fix and get help on various [automated validation tooling] failures on your [Azure REST API specs PR].
 
+For more help, see [aka.ms/azsdk/pr-getting-help] and [aka.ms/azsdk/support].
+
 ## Prerequisites
 
 Most guides here require for you to have `npm` installed, which you can get by installing [Node.js](https://nodejs.org/en/download).
 
-## Spell check
+## `Swagger SpellCheck`
```
Comment (weshaggard) [inline documentation/ci-fix.md:41]: OpenAPI?

File: documentation/ci-fix.md [hunk 2]
```diff
@@ -36,7 +66,7 @@ If you need more information on see [cspell configuration](https://cspell.org/co
 
 *Note*: We are trying to move away from one shared dictionary file so try and avoid editing custom-words.txt in the root as it will likely go away in the future.
 
-## Prettier check
+## `Swagger PrettierCheck`
```
Comment (weshaggard) [inline documentation/ci-fix.md:69]: These heading updates will likely break anchor links like `https://aka.ms/azsdk/specs/prettier` which is currently pointing at `https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/ci-fix.md#prettier-check`. So once these go in be sure to update the aka links.

File: documentation/ci-fix.md (hunk 1 above)
Comment (konrad-jamrozik) [reply documentation/ci-fix.md:41]: Matching checks name for now, on purpose.

File: documentation/ci-fix.md (hunk 2 above)
Comment (konrad-jamrozik) [reply documentation/ci-fix.md:69]: I found these:

image

**Review by weshaggard — APPROVED** [review]

**Pull request merged**